### DISCUSSION TITLE
Starter application template

### DISCUSSION THREAD

**Discussion by sinisaos** [piccolo-orm/piccolo#464] [Show and tell]
An example of a full stack project template that includes a few useful things for an easy start. Frontend is Vue app. 

Source code is [here](https://github.com/sinisaos/piccolo-vue-starter). Feel free to use it.

**Comment by dantownsend** [comment]
@sinisaos This looks great. Can we add it to https://github.com/piccolo-orm/piccolo_examples?

**Reply by sinisaos** [reply]
@dantownsend Thanks. Of course, it may be useful to someone.
